### ISSUE TITLE
Bug report: 

### ISSUE THREAD

**Issue by jiangshuyi02** [saturday06/VRM-Addon-for-Blender#541]
## Describe the bug

VRM cannot export，
TypeError: Cannot assign a 'IDPropertyGroup' value to the existing 'hops' Group IDProperty

## To Reproduce

1.Export vrm
2.See error 
Python: Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Roaming\Blender Foundation\Blender\4.2\extensions\blender_org\vrm\exporter\vrm0_exporter.py", line 145, in export_vrm
    self.mesh_to_bin_and_dict(progress.partial_progress(0.8))
  File "C:\Users\Administrator\AppData\Roaming\Blender Foundation\Blender\4.2\extensions\blender_org\vrm\exporter\vrm0_exporter.py", line 2240, in mesh_to_bin_and_dict
    mesh_data[prop] = mesh.data[prop]
    ~~~~~~~~~^^^^^^
TypeError: Cannot assign a 'IDPropertyGroup' value to the existing 'hops' Group IDProperty

## Expected behavior

hope it can be fix

## Screenshots

If applicable, add screenshots to help explain your problem.
![Snipaste_2024-08-09_14-22-40](https://github.com/user-attachments/assets/5b87d4e4-1bd0-4573-8167-a8c5175cb013)
![Snipaste_2024-08-09_14-23-57](https://github.com/user-attachments/assets/ced3868f-733e-49e1-98a7-6c198fc4ba7e)

## Desktop (please complete the following information)

- OS: [Windows]
- Blender [4.2]
- Addon Version [2.20.73]

## Additional context

Add any other context about the problem here.

**Comment by saturday06** [comment]
Thanks for your report!

The bug has been fixed in the latest version 2.20.76 just released. Please try it.

**Comment by jiangshuyi02** [comment]
Thank you very much, the bug has been resolved.
